### DISCUSSION TITLE
Release Cadence to Match Electron

### DISCUSSION THREAD

**Discussion by scottkuhl** [ElectronNET/Electron.NET#787] [Ideas]
I think it would be a good idea to establish a community policy to update Electron.NET to match the latest version of Electron.  As I write this, Electron.NET is running against 23 and Electron is on 25.

This would go a long way to making the project feel “alive”.

Creating a simple guide on this repo on the steps to update the host project, sample app, docs, tests etc. to run would allow any community member to pick it up and run with it.

**Comment by FlorianRappl** [comment]
Definitely. While the electron version can now be set I think more documentation on how to contribute would benefit.
